client/src/routes/Home/home.module.css

```diff
@@ -73,16 +73,30 @@
   font-size: var(--h6mobile);
   line-height: var(--lh-condensed-ultra);
   color: #9a9a9d;
-  padding: 10px 0;
   text-align: center;
   cursor: pointer;
+  transition: all 0.1s ease-in;
+}
+
+.listItem:hover {
+  color: var(--primary-color);
+  border-bottom: 5px solid var(--primary-color);
 }
 
 .listItemActive {
   composes: listItem;
   color: var(--primary-color);
   border-bottom: 5px solid var(--primary-color);
 }
+
+.tabButton {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  color: inherit;
+  padding: 10px 0;
+}
+
```

Comment:
Due eslint I was forced to create a button, because I couldn't apply onClick event to li